dev-python/flask-babelex/flask-babelex-0.9.4.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1
+
+MY_PN="Flask-BabelEx"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Adds i18n/l10n support to Flask applications"
+HOMEPAGE="https://github.com/mrjoes/flask-babelex https://pypi.org/project/Flask-BabelEx/"
+SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND="dev-python/flask[${PYTHON_USEDEP}]
+	>=dev-python/Babel-1[${PYTHON_USEDEP}]
+	>=dev-python/speaklater-1.2[${PYTHON_USEDEP}]
+	>=dev-python/jinja-2.5[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/sphinx[${PYTHON_USEDEP}]
+		dev-python/flask-sphinx-themes[${PYTHON_USEDEP}]
+	)"
+
+PATCHES=( "${FILESDIR}/${P}-tests-fix.patch" )
```

Comment:
The patch doesn't seem to be included.

Reply:
yes. you are right. included copy now. Also, there is already PR in upstream which fixes it: https://github.com/mrjoes/flask-babelex/pull/24

Hoping it to get merged and released soon.

Reply:
Why are you copying it? It doesn't seem that you've changed anything.

Reply:
I'm really hoping that next version bump will happen with test fixes which will mean that we'll just remove this file at all with the next version bump. Do I understand correctly that you are suggesting to hardcode usage to existing patch or rename existing patch to some generic name and use it?

Reply:
Just use the old name.